package/harvester-os/templates/91-harvester-bootstrap-repo.yaml

```diff
@@ -0,0 +1,39 @@
+bootstrapResources:
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: harvester-cluster-repo
+    namespace: cattle-system
+  spec:
+    selector:
+      matchLabels:
+        app: harvester-cluster-repo
+    replicas: 1
```

Comment:
Do we need HA for the repository, what if it's down during an upgrade?

Reply:
Just tested it, draining the node that contains the pod will re-create a new pod on another node.
So maybe it's not needed? We'll preload images on each node before upgrading Harvester.